torch/fx/graph.py

```diff
@@ -713,19 +727,14 @@ def print_tabular(self):
         print(tabulate(node_specs,
               headers=['opcode', 'name', 'target', 'args', 'kwargs']))
 
-    def lint(self, root : Optional[torch.nn.Module] = None):
+    def lint(self):
```

Comment:
This is technically BC-breaking, but we're in beta, so probably fine. The other alternative would be to accept the argument and print a deprecation warning